Kickstarter-iOS/Views/Controllers/PledgeDescriptionViewController.swift

```diff
@@ -1,45 +1,60 @@
 import Prelude
 import UIKit
 
 private enum Layout {
-  enum ImageView {
-    static let width: CGFloat = 90
-    static let height: CGFloat = 120
+  enum Description {
+    static let spacing: CGFloat = 10
   }
 
-  enum SpacerView {
-    static let height: CGFloat = 10
+  enum RewardThumbnail {
+    static let maxHeight: CGFloat = 130
   }
 }
 
 final class PledgeDescriptionViewController: UIViewController {
-  fileprivate let viewModel = PledgeDescriptionViewModel()
-
   // MARK: - Properties
 
-  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
-  private lazy var containerImageView: UIView = {
-    UIView(frame: .zero) |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  private lazy var dateLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var descriptionStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var estimatedDeliveryLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var expandIconImageView: UIImageView = { UIImageView(frame: .zero) }()
+  private lazy var learnMoreTextView: UITextView = { UITextView(frame: .zero) |> \.delegate .~ self }()
+  private lazy var longPressGestureRecognizer: UILongPressGestureRecognizer = {
+    let longPressGestureRecognizer = UILongPressGestureRecognizer(
+      target: self, action: #selector(PledgeDescriptionViewController.depress(_:))
+    )
+    longPressGestureRecognizer.minimumPressDuration = CheckoutConstants.RewardCard.Transition
+      .DepressAnimation.longPressMinimumDuration
+    longPressGestureRecognizer.delegate = self
+
+    return longPressGestureRecognizer
   }()
 
-  private lazy var pledgeImageView: UIImageView = {
-    UIImageView(frame: .zero) |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  internal lazy var rewardCardContainerShadowView: UIView = { UIView(frame: .zero) }()
+  internal lazy var rewardCardContainerMaskView: UIView = { UIView(frame: .zero) }()
+  private var rewardCardContainerShadowViewHeightConstraint: NSLayoutConstraint?
+  private var rewardCardContainerShadowViewWidthConstraint: NSLayoutConstraint?
+  internal lazy var rewardCardContainerView: RewardCardContainerView = {
+    RewardCardContainerView(frame: .zero)
+      |> \.delegate .~ self
+      |> \.isUserInteractionEnabled .~ false
   }()
 
-  private lazy var descriptionStackView: UIStackView = { UIStackView(frame: .zero) }()
-  private lazy var estimatedDeliveryLabel: UILabel = { UILabel(frame: .zero) }()
-  private lazy var dateLabel: UILabel = { UILabel(frame: .zero) }()
-  private lazy var spacerView = UIView(frame: .zero)
-  private lazy var learnMoreTextView: UITextView = { UITextView(frame: .zero) |> \.delegate .~ self }()
+  private let viewModel = PledgeDescriptionViewModel()
+
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
 
   // MARK: - Lifecycle
 
   override func viewDidLoad() {
     super.viewDidLoad()
 
     self.configureSubviews()
+
+    self.bindViewModel()
```

Comment:
I don't think we need to manually call this, should happen by default from calling `super.viewDidLoad()`

Reply:
Ah right thanks, just overhang from all the refactoring